app-misc/unfoo/unfoo-1.0.8-r2.ebuild

```diff
@@ -16,7 +16,10 @@ RESTRICT="!test? ( test )"
 COMMON_DEPS="
 	app-alternatives/bzip2
 	app-alternatives/gzip
-	app-arch/p7zip
+	|| (
+		>=app-arch/7zip-24.09[symlink(+)]
```

Comment:
I'm not sure whether Portage will prompt you to change the flag if you already have 7zip installed or just installed p7zip instead. Just worth bearing in mind.